### PR TITLE
Release v1.4.0 version

### PR DESCRIPTION
**Pull request by Sagilio**
Change the release note and clean up the project properties.

### CODE REVIEW

File: NetCasbin/NetCasbin.csproj [hunk 1]
```diff
@@ -3,28 +3,23 @@
...
+    <PackageReleaseNotes>1. Add Batch APIs，Scaling ABAC (eval function) features.
+2. Improve performance and reduce the allocate of enforcer and build in funtions.</PackageReleaseNotes>
+    <Authors>Casbin, Zhikui H</Authors>
+    <Company>Zhikui H</Company>
+    <Product>Casbin.NET</Product>
+    <RepositoryType>GIT</RepositoryType>
+    <Copyright>Apache License 2.0</Copyright>
     <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
-    <PackageReleaseNotes>1. Add KeyMatch4 function support</PackageReleaseNotes>
+    <RepositoryUrl>https://github.com/casbin-net/Casbin.NET</RepositoryUrl>
```
Comment (hsluoyz) [inline NetCasbin/NetCasbin.csproj:22]: The path should be: https://github.com/casbin/Casbin.NET

Comment (Sagilio) [reply NetCasbin/NetCasbin.csproj:22]: Ok

**Comment by hsluoyz** [comment]
@huazhikui @xcaptain

**Review by xcaptain — APPROVED** [review]
LGTM

**Comment by xcaptain** [comment]
Also please release a new version of EFcore-adapter to nuget, didn't update for a long time.

**Comment by hsluoyz** [comment]
Merged.

**Pull request merged**